src/Stripe.net/Entities/SubscriptionsScheduleRevisions/SubscriptionScheduleRevision.cs

```diff
@@ -0,0 +1,74 @@
+namespace Stripe
+{
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
+    public class SubscriptionScheduleRevision : StripeEntity, IHasId, IHasObject
+    {
+        /// <summary>
+        /// Unique identifier for the object.
+        /// </summary>
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// String representing the object’s type. Objects of the same type share the same value.
+        /// </summary>
+        [JsonProperty("object")]
+        public string Object { get; set; }
+
+        /// <summary>
+        /// Time at which the object was created. Measured in seconds since the Unix epoch.
+        /// </summary>
+        [JsonProperty("created")]
+        [JsonConverter(typeof(DateTimeConverter))]
+        public DateTime? Created { get; set; }
+
+        /// <summary>
+        /// The schedule's default invoice settings.
+        /// </summary>
+        [JsonProperty("invoice_settings")]
+        public SubscriptionScheduleInvoiceSettings InvoiceSettings { get; set; }
+
+        /// <summary>
+        /// Has the value <code>true</code> if the object exists in live mode or the value
+        /// <code>false</code> if the object exists in test mode.
+        /// </summary>
+        [JsonProperty("livemode")]
+        public bool Livemode { get; set; }
+
+        /// <summary>
+        /// Configuration for the subscription schedule’s phases.
+        /// </summary>
+        [JsonProperty("phases")]
+        public List<SubscriptionSchedulePhase> Phases { get; set; }
+
+        /// <summary>
+        /// TODO: Figure out docs
```

Comment:
Sure, I did this mostly because the compiler complained otherwise originally (it was expanded in my code). I'm not planning to keep as is, just missed it :)